cmd/podman/common/specgen.go

```diff
@@ -128,18 +130,21 @@ func getIOLimits(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string) (
 
 func getPidsLimits(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string) (*specs.LinuxPids, error) {
```

Comment:
It does not look like this function returns an error, should we drop the error?

Reply:
sure, pushed a new version.